plasma_framework/migrations/1041_setup_vault.js

```diff
@@ -103,8 +103,8 @@ module.exports = async (
     // eslint-disable-next-line no-unused-vars
     [deployerAddress, maintainerAddress, authorityAddress],
 ) => {
-    const vault = process.env.VAULT || false;
-    if (vault) {
+    const vault = process.env.VAULT === 'true';
+    if (vault === true) {
```

Comment:
Nitpick: I prefer the way it was.
`vault` is definitely a boolean now, and IMO `if (vault)` is better than `if (vault === true)`
It's not a strong opinion though, so leave it if you think `if (vault === true` is clearer